code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -1,6 +1,8 @@
 /obj/item/melee/energy
 	var/active = 0
 	var/force_on = 30 //force when active
+	var/force_off = 0 //Used to properly reset the force
+	var/throwforce_off = 0
```

Comment:
Since you don't intend for these to be set by the child datums, AND since you don't really care about initial value (you overwirite it in Init anyway), i suggest 1) moving them down the list of vars and 2) not doing the explicit `= 0`

Reply:
Done!